frontend/src/scenes/cohorts/Cohorts.tsx

```diff
@@ -182,6 +182,7 @@ export function Cohorts(): JSX.Element {
                     allowClear
                     enterButton
                     style={{ maxWidth: 400, width: 'initial', flexGrow: 1 }}
+                    autoFocus
```

Comment:
This LGTM, though I'm usually against auto-focused text inputs on scenes since it breaks mobile support ("why's the keyboard now open?"), and also breaks shortcut keys. 

We might eventually change all the defaults to something like `autoFocus={!isMobile()}`, but that's outside the scope here.

Reply:
Okay, I do much prefer being able to immediately search when I land on a page but seems fair!